rules.md

```diff
@@ -483,42 +482,43 @@ The following roles are required to follow Marine Law by the server rules. Break
 
 ### Exceptions
 - Military Police may ignore Minor Crimes unless ordered to enforce them, but they may not ignore Major or Capital Crimes.
-- Survivor Synthetics are not forced to follow Marine Law, dependent on their chosen alignment (Colony/Survivors/USCM/WY etc.)
+- Survivor Synthetics might not be forced to follow Marine Law, dependent on their chosen alignment. (Colony/Survivors/USCM/WY etc.)
 - Marine Law can be broken when necessary to preserve the lives and safety of others.
 
 ## Mutinies
 
-Mutinies must be Adminhelped by a chosen leader stating the reasoning and the players directly involved. There must be at least 5 members. with MPs not eligible to mutiny.
+Mutinies must be Adminhelped by a chosen leader stating the reasoning and the players directly involved. They should also follow the mutiny rules as outlined below.
 
+- Mutinies must have 5 members minimum.
+    - Roles bound to follow Marine Law through Mandatory Obedience are unable to participate in a mutiny. They ARE allowed to stay neutral if they agree with the mutineers though.
 - If the mutiny does not have a valid reason, it can't happen.
     - Staff reserve the right to grant or deny mutinies, and to decide if the reason given is sufficient.
-- Mutiny cannot occur during hijack, and any ongoing mutiny must be ended upon hijack.
+- Regular mutinies cannot occur during hijack, and any ongoing mutiny must be ended upon hijack.
 - Only the Acting Commander can be mutinied against for their decisions, actions, or inactions. They must be given a chance to stand down peacefully.
-- During the mutiny, attacking any uninvolved, medical, or surrendered personnel is strictly forbidden, as is perma-killing anyone involved. If you have surrendered during a mutiny, you must remain as such.
-    - Once you have picked a side, it is permanent. No take-backs unless you surrender.
+- During the mutiny attacking any uninvolved, medical or surrendered personnel is strictly forbidden, as is intentionally perma-killing anyone involved.
+    - Once you have picked a side, it is permanent. No flip flopping.
+    - Additionally, if you have surrendered during a mutiny you cannot restart hostilities and must follow regular escalation rules again.
```

Comment:
May be worth including a note here about medical personnel who have been engaging in active combat in the mutiny.